src/rashmi/NestedSwitch_assignments/FrequencyOfChar.java

```diff
@@ -0,0 +1,47 @@
+/*
+Assignment - 16 : 9th April'2021
+
+Find the frequency of character from a given String array.
+Note: Take String and character from the user.
+
+Input : technocredits
+char : e
+
+
+output : e -> 2*/
+
+package rashmi.NestedSwitch_assignments;
+
+import java.util.Scanner;
+
+public class FrequencyOfChar {
+
+	int count;
```

Comment:
You should write this instance variable with in method as, it’s useful for only that method.  If variable going to use multiple places then it makes senses to define that variable at instance. just trying to explain good coding practices.